.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "tools/mist2templateflow"]
 	path = tools/mist2templateflow
-	url = https://github.com/SIMEXP/mist2templateflow.git
```

Comment:
temporary switching to my fork to test with the fix for the nilearn version

Reply:
I have merged the version bump!

Reply:
will switch back top the upstream repo